shell/platform/windows/cursor_handler_unittests.cc

```diff
@@ -0,0 +1,27 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+#include "flutter/shell/platform/windows/cursor_handler.h"
+
+#include <memory>
+#include <vector>
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace flutter {
+namespace testing {
+TEST(CursorHandlerTest, CreateDummyCursor) {
+  // create 4x4 rawRGBA dummy cursor buffer
```

Comment:
```suggestion
  // Create 4x4 rawRGBA dummy cursor buffer.
```
Comments should be full sentences, starting with a capital letter and ending with a `.`. Among other places.